Sample_v3/Extensions/UITableViewController+Cells.swift

```diff
@@ -66,4 +67,24 @@ extension UITableViewController {
         
         return cell
     }
+    
+    func memberCell(_ member: ChatChannelMember, isCurrentUser: Bool) -> UITableViewCell {
+        let cell = tableView
+            .dequeueReusableCell(withIdentifier: "MemberCell") ?? .init(style: .default, reuseIdentifier: "MemberCell")
+        cell.textLabel?.text = createMemberNameAndStatusInfoString(for: member, isCurrentUser: isCurrentUser)
+        cell.textLabel?.textColor = member.name.flatMap(UIColor.forUsername) ?? .black
+        cell.textLabel?.font = .boldSystemFont(ofSize: cell.textLabel?.font.pointSize ?? UIFont.systemFontSize)
+        
+        cell.detailTextLabel?.text = createMemberOnlineStatusInfoString(for: member)
+        cell.detailTextLabel?.textColor = member.isOnline ? .blue : .lightGray
+
+        let banStateLabel = UILabel(frame: .init(x: 0, y: 0, width: 100, height: 40))
```

Comment:
nit: `banStatusLabel`, `state` means something different

Reply:
fixed in 9a22e8c091f2816e348792f97aed982af3afd5fe